frontend/post/postDirective.js

```diff
@@ -240,8 +240,9 @@
                         postCtrl.loadingPost = false;
                         const postAuthorPermissions = ["edit_post", "remove_post"];
                         postCtrl.user.addPermissions(postAuthorPermissions, response.key);
-                    }, function error() {
-                        AuthService.reload().then(function success() {
+                    }, function error(response) {
+                        UserService.load().then(function success(responseUser) {
```

Comment:
Why do you need this? And if you really need this, postCtrl.user = responseUser is overwriting the authService's user.